src/app/print/capture/capture-page.css

```
@@ -12,6 +12,11 @@ body {
   background: #fff;
 }
 
+@page {
+  size: A4;
```

Stylelint (17.14.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)